inc/functions/files.php

```diff
@@ -102,8 +102,9 @@ function rocket_generate_advanced_cache_file() {
 	}
 	$done = true;
 
+	// This filter is commented in inc/functions/formatting.php.
 	rocket_put_content(
-		rocket_get_constant( 'WP_CONTENT_DIR' ) . '/advanced-cache.php',
+		apply_filters( 'rocket_wp_content_dir', rocket_get_constant( 'WP_CONTENT_DIR' ) ) . '/advanced-cache.php',
```

Comment:
@Screenfeed Why do we need a filter here?

Reply:
I didn't added it myself, this comes from merging the branch `develop`: https://github.com/wp-media/wp-rocket/commit/42d5ac883530c9be50bb1989df62238eb1380557#diff-11ce37d5b1c39c6f26bb2a55bce5b5d1R106-R109.

Reply:
I added it initially as a way to correctly point `WP_CONTENT_DIR` to the Vfs, but you did it differently now, so it can be removed

Reply:
We'll remove it in a separate PR targeted to `develop`. Thanks @Screenfeed and @Tabrisrp

Reply:
for this specific occurrence, it was added in a commit in this PR: https://github.com/wp-media/wp-rocket/pull/2507/commits/53efc2f715c3e8849dfc7c2cafd3183cd3803e8c

so it can be removed here. The other one in `formatting.php` can be changed in a different PR though.

Reply:
Commit https://github.com/wp-media/wp-rocket/pull/2507/commits/1306e283d4abca9d21805e875cdd4e0253b40dc2 removes it from this PR.

PR #2602 removes it from `formatting.php` and the tests that used it.